### PR TITLE
Update ssl docs with instructions to invoke api using Postman

### PR DESCRIPTION
**Pull request by msm1992**
## Purpose
Update ssl docs with instructions to invoke api using Postman

related issue : https://github.com/wso2/product-apim/issues/6904

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: en/docs/Learn/APISecurity/APIAuthentication/secure-apis-using-mutual-ssl.md [hunk 1]
```diff
@@ -37,16 +37,16 @@ This section explains how to APIs in WSO2 API Manager can be secured using mutua
...
-    Before you begin...
-
-    Add the relevant certificate to your browser according to your private certificate.
-
-1.  [Invoke an API using the Integrated API Console](/Learn/ConsumeAPI/InvokeApis/InvokeApisUsingTools/invoke-an-api-using-the-integrated-api-console/) .
-2.  When you click **Execute** the browser will send a prompt similar to the one shown below. Select the corresponding certificate for the API.
-    ![]({{base_path}}/assets/attachments/103334944/103334940.png)
+1.  Import the certificte and private key to Postman. Navigate to certificates tab in Postman settings.
+    
+    ![](../../../assets/img/Learn/add-certificate-to-postman.png)
```
Comment (Mariangela) [inline en/docs/Learn/APISecurity/APIAuthentication/secure-apis-using-mutual-ssl.md]: Replace ../../../  with {{base_path}}/  please.

File: en/docs/Learn/APISecurity/APIAuthentication/secure-apis-using-mutual-ssl.md [hunk 2]
```diff
@@ -37,16 +37,16 @@ This section explains how to APIs in WSO2 API Manager can be secured using mutua
...
+    
+    Add the certificate and private key.
+    ![](../../../assets/img/Learn/provide-crt-and-private-key.png)
```
Comment (Mariangela) [inline en/docs/Learn/APISecurity/APIAuthentication/secure-apis-using-mutual-ssl.md]: Replace ../../../  with {{base_path}}/  please.

File: en/docs/Learn/APISecurity/APIAuthentication/secure-apis-using-mutual-ssl.md [hunk 3]
```diff
@@ -37,16 +37,16 @@ This section explains how to APIs in WSO2 API Manager can be secured using mutua
     
 6.  **Save** the API
     
-### Invoke an API secured with Mutual SSL from the Developer Portal
+### Invoke an API secured with Mutual SSL from Postman
```
Comment (Mariangela) [inline en/docs/Learn/APISecurity/APIAuthentication/secure-apis-using-mutual-ssl.md]: Replace "from" with "using"

File: en/docs/Learn/APISecurity/APIAuthentication/secure-apis-using-mutual-ssl.md (hunk 1 above)
Comment (msm1992) [reply en/docs/Learn/APISecurity/APIAuthentication/secure-apis-using-mutual-ssl.md]: fixed with 73351c9a4314b2fc29735a0767dc4e73f1197372

File: en/docs/Learn/APISecurity/APIAuthentication/secure-apis-using-mutual-ssl.md (hunk 2 above)
Comment (msm1992) [reply en/docs/Learn/APISecurity/APIAuthentication/secure-apis-using-mutual-ssl.md]: fixed with 73351c9a4314b2fc29735a0767dc4e73f1197372

File: en/docs/Learn/APISecurity/APIAuthentication/secure-apis-using-mutual-ssl.md (hunk 3 above)
Comment (msm1992) [reply en/docs/Learn/APISecurity/APIAuthentication/secure-apis-using-mutual-ssl.md]: fixed with 73351c9a4314b2fc29735a0767dc4e73f1197372

**Pull request merged**